### PR TITLE
chore: [IA-51] Fixes accessibility for sectionCardComponent

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR introduces an accessibility improvement on the SectionCardComponent to handle properly payment method refresh and bonuses refresh

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -2061,6 +2061,8 @@ faq:
     title: "My card has no security code."
     content: "Check well on the back (for Maestro, Mastercard, Visa) or on the front (for American Express cards). If you don't find any security code, most likely your card is not enabled for online payments. Contact your bank for more information."
 bonus:
+  accessibility:
+    sectionCardLabel: "Bonus and discounts"
```
Comment (fabriziofff) [inline locales/en/index.yml:2065]:
```suggestion
    sectionCardLabel: "Update bonus and discounts"
```
For the same reason of the previous comment.

File: locales/it/index.yml [hunk 2]
```diff
@@ -2089,6 +2089,8 @@ faq:
     title: "La mia carta non ha alcun codice di sicurezza."
     content: "Controlla bene sul retro (per le carte Maestro, Mastercard, Visa) o sul fronte (per le carte American Express). Se non trovi alcun codice di sicurezza, molto probabilmente la tua carta non è abilitata ai pagamenti online. Contatta la tua banca per maggiori informazioni."
 bonus:
+  accessibility:
+    sectionCardLabel: "Bonus e sconti"
```
Comment (fabriziofff) [inline locales/it/index.yml:2093]:
```suggestion
    sectionCardLabel: "Aggiorna bonus e sconti"
```
For the same reason of the previous comment.

File: locales/en/index.yml [hunk 3]
```diff
@@ -894,8 +894,8 @@ wallet:
     noName: Name unavailable
     loading: Loading bank list...
   accessibility:
-    sectionCardLabel: "Update payment methods"
-    sectionCardHint: "Update the list of payment methods"
+    sectionCardLabel: "Payment methods"
+    sectionCardHint: "double tap to update the list"
```
Comment (fabriziofff) [inline locales/en/index.yml:898]:
```suggestion
    sectionCardLabel: "Update payment methods"
    sectionCardHint: "Update the list of payment methods"
```

During the story https://pagopa.atlassian.net/browse/IA-47 we decided to change the text with "Update payment methods / Update the list of payment methods" in order to have a text that explains what the button do and additionaly give some other feedback in the hint, without giving explicit references to specific implicit actions, such as double taps.
Maybe that the story IA-51 it hasn't been updated after these changes, but it should reflect the same behaviour, what you think?

File: locales/it/index.yml [hunk 4]
```diff
@@ -913,7 +913,7 @@ wallet:
     loading: Carico la lista delle banche...
   accessibility:
     sectionCardLabel: "Aggiorna metodi di pagamento"
-    sectionCardHint: "Aggiorna la lista dei metodi di pagamento"
+    sectionCardHint: "tocca due volte per aggiornare la lista"
```
Comment (fabriziofff) [inline locales/it/index.yml:916]:
```suggestion
    sectionCardHint: "Aggiorna la lista dei metodi di pagamento"
```
For the same reason of the previous comment.

**Review by fabriziofff — CHANGES_REQUESTED** [review]

File: locales/en/index.yml (hunk 3 above)
Comment (CrisTofani) [reply locales/en/index.yml:898]: yes looks good!

**Review by fabriziofff — APPROVED** [review]

File: ts/components/wallet/card/SectionCardComponent.tsx [hunk 5]
```diff
@@ -194,56 +195,50 @@ const SectionCardComponent: React.FunctionComponent<Props> = (props: Props) => {
...
         >
           <View
-            style={[
-              styles.card,
-              styles.flatBottom,
-              cardStyle || styles.cardGrey
-            ]}
+            style={[styles.cardInner]}
+            accessibilityLabel={props.accessibilityLabel}
+            accessibilityHint={I18n.t("wallet.accessibility.sectionCardHint")}
```
Comment (fabriziofff) [inline ts/components/wallet/card/SectionCardComponent.tsx:218]: In this case shouldn't be something different, like "Aggiorna la lista dei bonus e sconti"? We have:
"Aggiorna bonus e sconti, pulsante, Aggiorna la lista dei metodi di pagamento".

**Review by fabriziofff — CHANGES_REQUESTED** [review]

File: ts/components/wallet/card/SectionCardComponent.tsx (hunk 5 above)
Comment (CrisTofani) [reply ts/components/wallet/card/SectionCardComponent.tsx]: my bad, forgot to spread the accessibility hint prop as for the label commit c34a4f8 implements it

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**